stl/inc/xlocale

```diff
@@ -1375,7 +1373,7 @@ private:
     _Codecvt_mode _Mode; // default: _Consume_header
 };
 
-#if defined(__cpp_char8_t) && !defined(_M_CEE_PURE)
+// Give me your power!

```

Comment:
This is not a very useful comment. Can we change it to something more informative

Reply:
we could also just get rid of this comment completely.